SMLHelper/Utility/MiscUtils.cs

```diff
@@ -6,7 +6,7 @@
     using System.IO;
     using UnityEngine;
 
-    public class Utility
+    public class MiscUtils
```

Comment:
@AlexejheroYTB  The usage of this class seems to indicate "PatchUtilities" or "PatchingUtilities" rather than anything miscellaneous.

@ahk1221 Question: Is everything being public here intentional?
If so, then it should probably be moved into the same `SMLHelper.V2.Utility` namespace.